src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -30,12 +30,26 @@ public static System.Diagnostics.Activity Current
         public string OperationName { get { throw null; } }
         public System.Diagnostics.Activity Parent { get { throw null; } }
         public string ParentId { get { throw null; } }
-        public ref System.Diagnostics.ActivitySpanId ParentSpanId { get { throw null; } }
+        public ref
+#if !NETSTANDARD1_3 // Could not get this to build with ref readonly returns. 
```

Comment:
I do not think that this will work.

If somebody builds against NetStandard1.3 and then runs against the newer version (netfx/net45/netcoreapp), they will get MissingMethodException. `readonly` modifier is modreq in the IL signature, and so it has to match between ref and def.

Reply:
How important is this property - would it be an option to just not have it in the .NET Standard 1.3 version of the public surface?

Reply:
I want to follow up off line on seeing if we can get this to compile for netstandard1.3, because clearly treating it specially is a hack, and it does not seem like there is a strong reason for it to have failed.  

The API is reasonably important, but it is not clear that netstandard1.3 is important (netstandard 2.0 and net46 are (these APIs are not in net45)).   That option may be viable but has risk.  

Note that this System.Diagnostics.DiagnosticSource package is stand-alone and does contain an implementation for the netandard1.3 framework, so it is not clear when you would actually get a mismatch between reference and implementation (since the implementation DLL is chosen at build time).

Reply:
> it is not clear when you would actually get a mismatch between reference and implementation 

1. Create library MyLib (NuGet package) that targets NS1.3 and calls this API in the library. The compiled code in this library will reference the API without `readonly`

2. Create program that targets netcoreapp and uses MyLib. When published, the program will have netcoreapp version of System.Diagnostics.DiagnosticSource. NuGet upgrades the reference, even though MyLib references NS1.3. However, MyLib will stay on NS1.3 because of there is no better version of it.	At the end, `System.Diagnostics.DiagnosticSource` in the published program will have the `readonly` API, but MyLib will be referencing the API without `readonly`.

Reply:
Jan was able to figure out how to get the Netstandard1.3 case working.   Thanks Jan.